pay-api/src/pay_api/models/base_model.py

```diff
@@ -70,34 +63,3 @@ def rollback():
     def find_by_id(cls, identifier: int):
         """Return model by id."""
         return cls.query.get(identifier)
-
-    @classmethod
-    def create_activity(cls, obj, is_delete=False):
-        """Create activity records if the model is versioned."""
```

Comment:
going to replace this with history tables, add in created_on, created_name etc updated_on etc to the tables